### ISSUE TITLE
Solution for updating gene and gene_alias tables

### ISSUE THREAD

**Issue by pieterlukasse** [cBioPortal/cbioportal#713]
We need a solution for keeping the gene and gene_alias tables up to date. How are you doing it now? Should we start a RFC for this?

**Comment by jjgao** [comment]
@pieterlukasse we usually update the gene and their aliases about twice a year when rebuilding the whole database. I also update other reference data such as pfam domains, structure mapping, etc in the same way.

I may not be easy to update gene without rebuilding the database.. The gene data is used to normalize genetic data when importing.

Unless you see a impressing need / use case / issue, I would not put this as a priority.

**Comment by pieterlukasse** [comment]
@jjgao : is this something we should want to automate? Would be nice to have a dedicated update script that does this for all reference data in cBioPortal. Or maybe add these features to [RFC3](https://docs.google.com/document/d/19O7qJIhISA_YPZ3EMx9Vpfo8Pc4C1Q8t8KzbSqW7jic/edit#heading=h.lr5cv6d6d547)?

**Comment by jjgao** [comment]
@pieterlukasse: it might deserve a separate RFC. GenomeNexus only deals with variant data. There are more reference data in cBioPortal.

As for the gene info data, I am concerned that without re-importing all genetic data, it may also cause inconsistency in the data to update the gene info only. 

For example, MLL2 was replaced by KMT2D. Imaging that we imported some copy number data before that change. Then we update the genes. Now the problem is MLL2 is also an alias of KMT2B. So after update gene info, the MLL2 data will not be able to correctly imported because of ambiguity. 

Please note that in the staging files we always need to deal with aliases / old symbols such as MLL2. I know even if we build a fresh database after updating gene info, we still cannot solve ambiguities like MLL2, but at least it's consistent, and we can try to solve it in a consistent way.

In short, I don't think there is a big need yet, but it's a good idea to keep it in the radar.

**Comment by pieterlukasse** [comment]
I found a script which works for genes (#805), but it needs some documentation and improvements to make it a script that updates the whole DB.

**Comment by pieterlukasse** [comment]
this has been since documented in https://github.com/cBioPortal/cbioportal/blob/master/docs/Updating-gene-and-gene_alias-tables.md and linked in https://github.com/cBioPortal/datahub/blob/master/seedDB/Update-Seed-Database.md
